tests/Google/Ads/GoogleAds/Lib/V1/LogMessageFormatterTest.php

```diff
@@ -233,17 +257,48 @@ private function createSuccessRequestResponse()
         ];
         $response = new SearchGoogleAdsResponse(['results' => $googleAdsRows]);
 
-        $unaryCall = $this->getMockBuilder(UnaryCall::class)
+        $call = $this->getMockBuilder(UnaryCall::class)
             ->disableOriginalConstructor()
             ->disableOriginalClone()
             ->disableArgumentCloning()
             ->getMock();
-        $unaryCall->method('getMetadata')->willReturn(
+        $call->method('getMetadata')->willReturn(
             ['request-id' => ['AbCdEfGh'], 'x-google-session-info' => ['1234abcd']]
         );
 
         $this->successRequest = compact('method', 'argument', 'metadata');
-        $this->successResponse = compact('status', 'response', 'unaryCall');
+        $this->successResponse = compact('status', 'response', 'call');
```

Comment:
ditto if you want to change for `call` instead of `unaryCall` (multiple times in this file)

Reply:
If you mean updating GoogleAdsLoggingUnaryCall, it's done. All mentioning of 'unaryCall' is changed to 'call' in this file and that of V2 already.
If not, please tell me again.